hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/SecureBulkLoadManager.java

```diff
@@ -341,27 +342,36 @@ private User getActiveUser() throws IOException {
     return user;
   }
 
-  private static class SecureBulkLoadListener implements BulkLoadListener {
+  protected static class SecureBulkLoadListener implements BulkLoadListener {
     // Target filesystem
     private final FileSystem fs;
     private final String stagingDir;
     private final Configuration conf;
     // Source filesystem
     private FileSystem srcFs = null;
     private Map<String, FsPermission> origPermissions = null;
+    private Map<String, String> origlSources = null;
```

Comment:
Could be <String,Path> and save need for path conversion again in `failedBulkLoad`?

Reply:
nit: `origSources` instead of `origlSources`?

Reply:
I left it like that because `origPermissions` uses String as well so a conversion would be done anyway. But now that I think about it there is no reason to keep `origPermissions` as String either. I could move both to use Path instead.

Reply:
> I left it like that because `origPermissions` uses String as well so a conversion would be done anyway. And while I could switch both of those to use Paths, log messages will need a string too.

Ok, let's just then follow var naming pattern, then.